pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[build-system]
+requires = ["setuptools>=61.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ppafm"
+dynamic = ["version"]
+description = "Classical forcefiled model for simulation atomic force microscopy images."
+authors = []
+readme = "README.md"
+license = {file = "LICENSE"}
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Science/Research",
+    "Natural Language :: English",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3",
+]
+requires-python = ">=3.6"
+dependencies = [
```

Comment:
We also depend on `matplotlib` in the main code, in the OpenCL code we depend on `pyopencl` and `reikna`, and for the GUI there is an optional dependency on `ase` for viewing the molecule geometry.

Reply:
Ok, I will put everything into the `dependencies`. We can split those things later if needed.